package.json

```diff
@@ -11,7 +11,12 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@nextui-org/button": "^2.0.38",
+    "@nextui-org/react": "^2.4.8",
+    "@nextui-org/system": "^2.2.6",
+    "@nextui-org/theme": "^2.2.11",
     "clsx": "^2.1.1",
+    "framer-motion": "^11.9.0",
```

Comment:
I don't see where you are using framer motion 
please delete it
we should avoid having unused libraries in out project

Reply:
Yeah, you’re totally right. I didn’t pay attention to it. It’s fixed [3d725c4](https://github.com/dark-orga/anitube-website/pull/1/commits/3d725c46fa85abddd21998c8daaeb3203318139b)